docs/documentation/stories/moving-into-the-cli.md

```diff
@@ -0,0 +1,63 @@
+# Moving your project to Angular-CLI
```

Comment:
Fixed.

@wardbell I don't see where 'we' is misused here. Can you point me in the right direction?

Regarding folder, they shouldn't be in the same folder. I'll clarify.